backend/src/bundles/user-details/user-details.controller.ts

```diff
@@ -0,0 +1,326 @@
+ *          id:
+ *            format: uuid #Example: '550e8400-e29b-41d4-a716-446655440000'
+ *            type: string
+ *          userId:
+ *            format: uuid #Example: '550e8400-e29b-41d4-a716-446655440000'
+ *            type: string
+ *          isApproved:
+ *            type: boolean
+ *          deniedReason:
+ *            type: string
+ *          isHired:
+ *            type: boolean
+ *          profileName:
+ *            type: string
+ *          salaryExpectation:
+ *            type: number
+ *          hiredSalary:
+ *            type: number
+ *          jobTitle:
+ *            type: string
+ *          location:
+ *            type: string
+ *          experienceYears:
+ *            type: number
+ *          employmentType:
+ *            type: array
+ *            items:
+ *              type: string
+ *          description:
+ *            type: string
+ *          englishLevel:
+ *            type: string
+ *          notConsidered:
+ *            type: array
+ *            items:
+ *              type: string
+ *          preferredLanguages:
+ *            type: array
+ *            items:
+ *              type: string
+ *          projectLinks:
+ *            type: array
+ *            items:
+ *              type: string
+ *          photoId:
+ *            type: string
+ *          fullName:
+ *            type: string
+ *          phone:
+ *            type: string
+ *          linkedinLink:
+ *            type: string
+ *          companyName:
+ *            type: string
+ *          companyLogoId:
+ *            type: string
+ *          companyWebsite:
+ *            type: string
+ *          employerPosition:
+ *            type: string
+ *          cvId:
+ *            type: string
+ */
+class UserDetailsController extends ControllerBase {
+    private userDetailsService: UserDetailsService;
+
+    public constructor(logger: Logger, userDetailsService: UserDetailsService) {
+        super(logger, ApiPath.USER_DETAILS);
+
+        this.userDetailsService = userDetailsService;
+
+        this.addRoute({
+            path: UserDetailsApiPath.UPDATE,
+            method: 'PATCH',
+            validation: {
+                body: userDetailsUpdateValidationSchema,
+            },
+            handler: (options) =>
+                this.update(
+                    options as ApiHandlerOptions<{
+                        body: UserDetailsUpdateRequestDto;
+                    }>,
+                ),
+        });
+
+        this.addRoute({
+            path: UserDetailsApiPath.APPROVE,
+            method: 'PATCH',
+            validation: {
+                body: userDetailsApproveValidationSchema,
+            },
+            preHandler: adminRoute,
+            handler: (options) =>
+                this.approve(
+                    options as ApiHandlerOptions<{
+                        body: UserDetailsApproveRequestDto;
+                    }>,
+                ),
+        });
+    }
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !

Reply:
Can I leave it in this task? Because it has already implemented.